package.json

```diff
@@ -7,7 +7,10 @@
     "doc": "doc"
   },
   "scripts": {
-    "build": "npm run collect && npm run bundle",
+    "build": "npm run postinstall && npm run collect && npm run bundle",
+    "build-design-tokens": "npm --prefix open-inwoner-design-tokens run build",
+    "watch-design-tokens": "npm --prefix open-inwoner-design-tokens run start",
+    "postinstall": "npm --prefix ./open-inwoner-design-tokens install && npm --prefix ./open-inwoner-design-tokens run build",
```

Comment:
Just curious: is there a difference between using `-- prefix ./open-inwoner-design-tokens` and `--prefix open-inwoner-design-tokens`?

Reply:
Ah, well spotted - yes there is a difference here: 

- **_with_** the dot (absolute path) it specifies an **absolute** path to the directory where NPM should look for the package's package.json file. In this case, it points to the current directory (./) and then to the open-inwoner-design-tokens directory. 
- **_without_** the dot (relative path) it assumes that the open-inwoner-design-tokens directory is present in the same directory where the NPM command is run, in our case in the root; 

I'll make this consistent.